web-frontend/src/pages/TransferCoins/SendCoins/Send.tsx

```diff
@@ -0,0 +1,149 @@
+  const [recipient, setRecipient] = useState('');
+  const [valid, setValid] = useState<boolean>(false);
+  const [error, setError] = useState<object | null>(null);
+  const [modal, setModal] = useState<boolean>(false);
+  const [modalAccounts, setModalAccounts] = useState<boolean>(false);
+
+  function validate(e: any) {
+    const formObject = parseForm(e);
+    const { amount, recipient } = formObject;
+    const amountNum = reverseFormatStandard(amount);
+
+    if (amountNum > balance) {
+      setError({ amount: Intl.t('errors.send.amount-to-high') });
+      return false;
+    }
+    if (amountNum <= 0) {
+      setError({ amount: Intl.t('errors.send.amount-to-low') });
+      return false;
+    }
+    if (Number.isNaN(amountNum)) {
+      setError({ amount: Intl.t('errors.send.invalid-amount') });
+      return false;
+    }
+    if (!recipient) {
+      setError({ recipient: Intl.t('errors.send.no-recipient') });
+      return false;
+    }
+    if (!checkRecipientFormat(recipient)) {
+      setError({ recipient: Intl.t('errors.send.invalid-recipient') });
+      return false;
+    }
+    setError(null);
+    return true;
+  }
+  function handleSubmit(e: FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+    if (!validate(e)) return;
+    else setValid(!valid);
+  }
+  const formattedBalance = formatStandard(balance, Unit.NanoMAS, 2);
+
+  function overrideAmount(pct: number) {
+    const newAmount = balance * pct;
+    const newFormatedAmount = formatStandard(newAmount, Unit.NanoMAS, 9);
+    setAmount(newFormatedAmount);
+  }
+
+  function SendPercentage({ percentage }: { percentage: number }) {
+    return (
+      <li
+        onClick={() => {
+          overrideAmount(percentage / 100);
+        }}
+        className="mr-3.5 hover:cursor-pointer"
+      >
+        {percentage === 100 ? 'Max' : `${percentage}%`}
+      </li>
+    );
+  }
+
+  function handleChange(e: ChangeEvent<HTMLInputElement>): void {
+    e.preventDefault();
+    const candidateAmount = e.target.value;
+    // if there is more than 9 decimals, set the amount to the previous value
+    if (candidateAmount.split('.')[1]?.length > 9) {
+      setAmount(amount);
+      return;
+    }
+    setAmount(e.target.value);
+  }
+
+  function handleFees(num: number) {
+    setFees(num);
+  }
+
+  function handleModal() {
+    setModal(!modal);
+  }
+
+  function handleModalAccounts() {
+    setModalAccounts(!modalAccounts);
+  }
+
+  const confirmArgs = {
+    amount,
+    nickname,
+    recipient,
+    valid: valid,
+    fees: fees,
+    setValid,
+  };
+
+  const sendArgs = {
+    amount,
+    account,
+    formattedBalance,
+    recipient,
+    error,
+    fees,
+    modal,
+    modalAccounts,
+    setModal,
+    handleModal,
+    setModalAccounts,
+    handleModalAccounts,
+    setFees,
+    handleFees,
+    setRecipient,
+    handleSubmit,
+    handleChange,
+    SendPercentage,
+  };
+
+  return (
+    <div className="mt-3.5">
```

Comment:
Please, no margin-top! It's the upper element that pushes down! The left that pushes right!

Reply:
yes but in this cas thhere is no upper element

Reply:
there is no upper elemnt it is passed as props in the SendCoins tabs component which does not allow encapsuled element like 

`
<div className="pb-3.5">
<Send/>
</div
`